src/js-tokens.ts

```diff
@@ -0,0 +1,80 @@
+import jsTokens, { type Token as JSToken } from 'js-tokens'
+import type { StripLiteralOptions } from './types'
+
+export function _stripLiteralJsTokens(code: string, options?: StripLiteralOptions) {
+  const FILL = options?.fillChar ?? ' '
+  const FILL_COMMENT = ' '
+  let result = ''
+
+  const filter = options?.filter ?? (() => true)
+
```

Comment:
Acorn implementation has `fillupTo`, but I am not sure how it is used to be honest 🤔

Reply:
In acorn AST the tokens could be discrete, we use that to fill with the original code between the gaps of two tokens. If js-tokens does not need it then no worries on it